packages/storage/compat/index.ts

```diff
@@ -45,12 +46,10 @@ function factory(
   const app = container.getProvider('app').getImmediate();
```

Comment:
Eventually we should get `app-compat` and pass it to `StorageServiceCompat `. Can you please add a `TODO` comment?

Reply:
Added.